README.md

```diff
@@ -1,8 +1,13 @@
-ELK autopilot pattern
+Autopilot Pattern ELK
 ==========
 
 *[Autopilot pattern](http://autopilotpattern.io/) implementation of ELK*
 
+[![DockerPulls](https://img.shields.io/docker/pulls/autopilotpattern/mysql.svg)](https://registry.hub.docker.com/u/autopilotpattern/mysql/)
```

Comment:
I'm an idiot, or a robot.

Thanks for pointing out my error. Fixed by adding badges for all the main images.